packages/app/src/domain/vaccine/components/vaccine-delivery-and-administrations-tooltip.tsx

```diff
@@ -90,8 +90,7 @@ export function VaccineDeliveryAndAdministrationsTooltip({
           </TooltipListItem>
         ))}
 
-        <Spacer mb={1} />
-        <TooltipListItem>
+        <TooltipListItem mt={1}>
```

Comment:
it's a styled component so you could go for the following and keep the TooltipListItem simple:
```suggestion
        <TooltipListItem css={css({{ mt: 1 })}>
```